lua/SinglePlayerLaunch.lua

```
@@ -122,6 +122,7 @@ function SetupCampaignSession(scenario, difficulty, inFaction, campaignFlowInfo,
     Prefs.SetToCurrentProfile('LoadingFaction', faction)
 
     sessionInfo.scenarioMods = import("/lua/mods.lua").GetCampaignMods(sessionInfo.scenarioInfo)
+    sessionInfo.scenarioMods += import("/lua/mods.lua").GetUiMods(sessionInfo.scenarioInfo)
```

Luacheck (1.2.0)

[error] assignment uses compound operator +=

(E033)